corehq/apps/locations/ucr_expressions.py

```diff
@@ -86,6 +86,8 @@ def __call__(self, item, context=None):
         location_id = self._location_id_expression(item, context)
         location_type = self._location_type_expression(item, context)
         location = self._get_ancestors_by_type(location_id, context).get(location_type)
+        if not location:
```

Comment:
Don't have much context on how reports use this. How would this even happen? that the parent location won't exist?
If it's always a user making an error in configuration, i agree on letting the user know instead of handling it silently.
If the location is not present anymore/was deleted so fine to ignore then.